packages/vite/src/node/plugins/html.ts

```diff
@@ -267,6 +267,7 @@ export function buildHtmlPlugin(config: ResolvedConfig): Plugin {
     },
 
     async generateBundle(_, bundle) {
+      const analyzedChunk: Map<any, any> = new Map()
```

Comment:
**suggestion (non-blocking, dx):** We should try to avoid `any` and use at least `unknown` or better typedefs.

Reply:
A little suggestion `const analyzedChunk: Map<OutputChunk, number> = new Map();`

Reply:
This is a good suggestion